### PR TITLE
fixups

### PR DESCRIPTION
**Pull request by lynxis**

### CODE REVIEW

File: broker/eventloop.py [hunk 1]
```diff
@@ -47,12 +47,17 @@ def start(self):
...
+                for fd, event in self.poller.poll():
+                    mapping = self.pollables.get(fd, None)
+                    if not mapping:
+                        continue
+
+                    pollable, file_object = mapping
+
+                    if event & select.EPOLLIN:
+                        pollable.read(file_object)
+            # IOError get produced by signal even. in version 3.5 this is fixed an the poll retries
```
Comment (kostko) [inline broker/eventloop.py]: Put the comment inside the `except` block.

File: broker/tunnel.py [hunk 2]
```diff
@@ -274,7 +277,7 @@ def close(self, reason=protocol.ERROR_REASON_UNDEFINED):
         :param reason: Reason code for the tunnel being closed
         """
 
-        logger.info("Closing tunnel %d." % self.tunnel_id)
+        logger.info("Closing tunnel %d after %d seconds", self.tunnel_id, time.time()-self.created_time)
```
Comment (kostko) [inline broker/tunnel.py]: Spaces around `-` operator.

File: client/l2tp_client.c [hunk 3]
```diff
@@ -763,10 +763,10 @@ void context_send_setup_request(l2tp_context *ctx)
   buf += uuid_len;
 
   // And the local tunnel identifier at the end
-  put_u16(&buf, ctx->tunnel_id);
+  put_u32(&buf, ctx->tunnel_id);
 
   // Now send the packet
-  context_send_packet(ctx, CONTROL_TYPE_PREPARE, (char*) &buffer, uuid_len + 9);
+  context_send_packet(ctx, CONTROL_TYPE_PREPARE, (char*) &buffer, (buf-&buffer[0]));
```
Comment (kostko) [inline client/l2tp_client.c]: Spaces around `-` operator.

**Comment by kostko** [comment]
Thanks, looks good besides some minor style issues.

**Comment by lynxis** [comment]
done.

**Comment by mitar** [comment]
BTW, how did you discover this one-off error? Were there any observable issues?

**Comment by lynxis** [comment]
@mitar the uuid is was wrong in the server log.

**Pull request merged**